backend/pkg/abstraction/sublogger.go

```diff
@@ -0,0 +1,9 @@
+package abstraction
+
+import "time"
+
+type Sublogger interface {
+	Log(timestamp time.Time, message string)
+	Start()
+	Stop()
+}
```

Comment:
Instead of adding a new ´Sublogger´ interface, add a `Start` and `Stop` methods to the overall `Logger` abstraction and use it both for the logger and the sub-loggers